src/main/java/sirius/biz/codelists/CodeListImportJob.java

```diff
@@ -0,0 +1,158 @@
+ * <p>
+ * If a language is provided, this job creates {@link sirius.biz.translations.Translation translations} accordingly.
+ *
+ * @param <E> the effective entity type used to represent code list entries
+ */
+public class CodeListImportJob<E extends BaseEntity<?> & CodeListEntry<?, ?, ?>> extends EntityImportJob<E> {
+    @Part
+    private static CodeLists<?, ?, ?> codeLists;
+
+    protected static final String CLE_VALUE =
+            CodeListEntry.CODE_LIST_ENTRY_DATA.inner(CodeListEntryData.VALUE).getName();
+    protected static final String CLE_ADDITIONAL_VALUE =
+            CodeListEntry.CODE_LIST_ENTRY_DATA.inner(CodeListEntryData.ADDITIONAL_VALUE).getName();
+    protected static final String CLE_DESCRIPTION =
+            CodeListEntry.CODE_LIST_ENTRY_DATA.inner(CodeListEntryData.DESCRIPTION).getName();
+
+    private String languagePrefix;
+
+    private final CodeList codeList;
+    private final Optional<String> languageParameter;
+
+    /**
+     * Creates a new job for the given factory, name and process.
+     *
+     * @param fileParameter        the parameter which is used to derive the import file from
+     * @param ignoreEmptyParameter the parameter which is used to determine if empty values should be ignored
+     * @param importModeParameter  the parameter which is used to determine the {@link ImportMode} to use
+     * @param codeListParameter    the parameter which specifies the target CodeList
+     * @param languageParameter    the parameter which specifies the language for which the CodeList should be imported
+     * @param type                 the type of entities being imported
+     * @param dictionary           the import dictionary to use
+     * @param process              the process context itself
+     * @param factoryName          the name of the factory which created this job
+     */
+    @SuppressWarnings("squid:S00107")
+    @Explain("We rather have 9 parameters here and keep the logic properly encapsulated")
+    public CodeListImportJob(FileParameter fileParameter,
+                             BooleanParameter ignoreEmptyParameter,
+                             EnumParameter<ImportMode> importModeParameter,
+                             CodeListParameter codeListParameter,
+                             Optional<String> languageParameter,
+                             Class<E> type,
+                             ImportDictionary dictionary,
+                             ProcessContext process,
+                             String factoryName) {
+        super(fileParameter, ignoreEmptyParameter, importModeParameter, type, dictionary, process, factoryName);
+        this.codeList = process.require(codeListParameter);
+        this.languageParameter = languageParameter;
+        languageParameter.ifPresent(lang -> {
+            this.languagePrefix = lang + "_";
+            this.dictionary.addField(FieldDefinition.stringField(languagePrefix + CLE_VALUE)
```

Comment:
this feels like it belongs in the import handler

Reply:
This also prevents unnecessary code duplication (see below)

Reply:
To prevent code duplication, it has been agreed to keep the code in this class.